server/src/storage/segment.rs

```diff
@@ -33,20 +33,20 @@ use crate::storage::data_point::DataPoint;
 use crate::storage::{BUILDER_CAPACITY, StorageEngine};
 use crate::types::{Timestamp, TimestampArray, TimestampBuilder, Value, ValueBuilder};
 
-/// Shared functionality between different types of uncompressed segments, such as segment builders
-/// and spilled segments.
+/// Shared functionality between different types of uncompressed segments, such as [`SegmentBuilder`]
+/// and [`SpilledSegment`].
 pub trait UncompressedSegment {
     fn get_record_batch(&mut self) -> Result<RecordBatch, ParquetError>;
 
     fn get_memory_size(&self) -> usize;
 
-    // Since both segment builders and spilled segments are present in the compression queue, both
+    // Since both SegmentBuilders and SpilledSegments are present in the queue of finished segments, both
```

Comment:
Maybe replace the instances of "finished segment" with "FinishedSegment".

Reply:
Fixed.